ddl/ddl_db_test.go

```diff
@@ -162,8 +162,6 @@ func (s *testDBSuite) TestMySQLErrorCode(c *C) {
 	s.testErrorCode(c, sql, tmysql.ErrUnknownCharacterSet)
 	sql = "create table t1(a int) character set laitn1;"
 	s.testErrorCode(c, sql, tmysql.ErrUnknownCharacterSet)
-	sql = "create table t2(c1.c2 blob default null);"
-	s.testErrorCode(c, sql, tmysql.ErrWrongTableName)
```

Comment:
Why remove this case?

Reply:
Added them back. They were accidentally deleted from codebase and nothing to do with this PR.